verifier/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM ubuntu:20.04
+
+LABEL maintainer="yishangzhang <Yishang.Zhang@linux.alibaba.com>"
+
+SHELL ["/bin/bash", "-c"]
+RUN apt-get update && \
+    apt-get install -y libjson-c4 libjson-c-dev mariadb-server libmysqlclient-dev git make gcc wget && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN mkdir /root/ibmtpm && \
+    cd  /root/ibmtpm && \
+    wget https://sourceforge.net/projects/ibmswtpm2/files/ibmtpm1682.tar.gz && \
+    tar zxvf ibmtpm1682.tar.gz && \
+    cd src && make install  
+
+RUN mkdir /root/tpm2 && \
+    cd /root/tpm2 && \
+    wget https://versaweb.dl.sourceforge.net/project/ibmtpm20tss/ibmtss1.6.0.tar.gz && \
+    tar zxvf ibmtss1.6.0.tar.gz
+
+RUN for i in {1..5}; do \
+        git clone https://github.com/kgoldman/acs.git /root/acs && \
+        break; \
+        sleep 5; \
+    done
+
+WORKDIR /usr/include 
+RUN ln -s json-c json
```

Comment:
please remove line 27-28 to lin 9 and use `ln -s /usr/include/json-c /usr/include/json `